superset/assets/stylesheets/less/cosmo/variables.less

```diff
@@ -21,15 +21,15 @@
 // --------------------------------------------------
 
 
-//== Colors
+// == Colors
 //
-//## Gray and brand colors for use across Bootstrap.
+// ## Gray and brand colors for use across Bootstrap.
 
 @gray-base:              #000;
 @gray-darker:            lighten(@gray-base, 13.5%);
 @gray-dark:              lighten(@gray-base, 20%);
-@gray:                   lighten(@gray-base, 33.5%);
-@gray-light:             lighten(@gray-base, 70%);
+@bs-gray:                   lighten(@gray-base, 33.5%);
+@bs-gray-light:             lighten(@gray-base, 70%);
```

Comment:
nit: tab miss aligned

Reply:
Yep, didn't worry about it since Prettier is about to wipe that out, and my next PR will rewrite or remove this file entirely ;)